pkg/prompts/prompts.go

```diff
@@ -244,3 +272,40 @@ func Select[T any](label string, items []T, opt *SelectOpt[T]) (T, error) {
 
 	return items[i], nil
 }
+
+func getCurrentDirName() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("getting current directory: %v", err)
+	}
+	return filepath.Base(dir), nil
+}
+
+// Sanitize the directory name to comply with k8s label rules
+func sanitizeAppName(name string) string {
```

Comment:
we need unit tests. Unit testing this function is a good start

Reply:
yeah, I'm working on the unit tests! I wanted to get initial thoughts so pushed without unit tests.